tomcat/src/main/docker/15-debug-env-tomcat.bash

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Adds bits of configuration necessary for the Cloud Debugger to work with the Tomcat image.
+# It needs to run before 20-debug-env.bash from the openjdk image.
+
+# Set CDBG_APP_WEB_INF_DIR, used by CDBG in format-env-appengine-vm.sh
+export CDBG_APP_WEB_INF_DIR="${CATALINA_BASE}/webapps/root/WEB-INF"
```

Comment:
`root` -> `ROOT`

Reply:
Ah yea  wasn't aware that tomcat unpacked it at all. Fixed